src/Microsoft.AspNetCore.Mvc.Razor.Extensions/InstrumentationPass.cs

```diff
@@ -101,14 +101,11 @@ public override void VisitCSharpExpression(CSharpExpressionIRNode node)
                 VisitDefault(node);
             }
 
-            public override void VisitExecuteTagHelpers(ExecuteTagHelpersIRNode node)
+            public override void VisitTagHelper(TagHelperIRNode node)
```

Comment:
We now instrument around the entire tag helper instead of just the execute block.